server/zanata-war/src/main/java/org/zanata/config/JsfModuleConfig.java

```diff
@@ -1,38 +0,0 @@
-/*
- * Copyright 2015, Red Hat, Inc. and individual contributors as indicated by the
- * @author tags. See the copyright.txt file in the distribution for a full
- * listing of individual contributors.
- *
- * This is free software; you can redistribute it and/or modify it under the
- * terms of the GNU Lesser General Public License as published by the Free
- * Software Foundation; either version 2.1 of the License, or (at your option)
- * any later version.
- *
- * This software is distributed in the hope that it will be useful, but WITHOUT
- * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
- * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
- * details.
- *
- * You should have received a copy of the GNU Lesser General Public License
- * along with this software; if not, write to the Free Software Foundation,
- * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA, or see the FSF
- * site: http://www.fsf.org.
- */
-package org.zanata.config;
-
-import javax.enterprise.inject.Specializes;
-
-/**
- * Specialization of DeltaSpike's JSFModule config
- * @author Carlos Munoz <a href="mailto:camunoz@redhat.com">camunoz@redhat.com</a>
- */
-@Specializes
-public class JsfModuleConfig extends
-        org.apache.deltaspike.jsf.api.config.JsfModuleConfig {
-    @Override
-    public boolean isInitialRedirectEnabled() {
-        // without a valid dswid (added by the redirect), FacesMessages
-        // can't find the correct window scope
-        return true;
```

Comment:
I could have removed the class when I added that comment and changed the value to `true`, because the `super` implementation in DeltaSpike is the same.